images/win/Windows2016-Readme.md

```diff
@@ -13,7 +13,7 @@
 - Go 1.15.15
 - Julia 1.7.2
 - Kotlin 1.6.10
-- LLVM 13.0.1
+- LLVM 14.0.0
```

Comment:
Don't update README, it's generated automatically

Reply:
Thanks for the info, didn't know that.